links.html

```diff
@@ -15,25 +19,27 @@
             <li> <a href="index.html">Home</a>  </li>
             <li> <a href="links.html">Links </a>  </li>
             <li> <a href="glossary.html">Glossary </a>  </li>
-            <li> <a href="https://jigsaw.w3.org/css-validator/"target = "blank">W3.org</a>  </li>
-            <li> <a href="https://developer.mozilla.org/en-US/"target = "blank">MDN</a>  </li>
-            <li> <a href="https://caniuse.com/"target = "blank">caniuse</a>  </li>
-            <li> <a href="https://necolas.github.io/normalize.css/8.0.1/normalize.css"target = "blank">Normalized CSS</a>  </li>
```

Comment:
wrong location for these links
incorrect target attribute (`blank` does not do the same thing as `_blank`)